client/src/API/get.ts

```diff
@@ -1,84 +1,116 @@
-import axios from 'axios';
-import { GeolocationCoordinates, LocationData } from 'types';
+import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios';
+import {
+  AuthenticationResponse,
+  CurrentWeatherData,
+  DailyWeather,
+  GeolocationCoordinates,
+  HourlyWeather,
+  LocationByQuery,
+  LocationData
+} from 'types';
 
-const URL = 'http://localhost:5000';
+const forecaClient: AxiosInstance = axios.create({
+  baseURL: 'http://localhost:5000',
+  headers: {}
+});
 
-export const createToken = async () => {
+forecaClient.interceptors.request.use(
+  async (config: AxiosRequestConfig) => {
+    const accessToken: string | null | undefined = localStorage.getItem('token');
+    if (accessToken && accessToken !== 'undefined') {
```

Comment:
please fix setting token in local storage (so it couldn't save 'undefined' string as value) and remove ` accessToken !== 'undefined'` condition